### PR TITLE
hotfix for ip address typo.

### PR DESCRIPTION
**Pull request by Seeker1911**

### CODE REVIEW

File: rest-api/config/config_prod.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
       "whitelisted_ip_ranges": {
         "ip4": [
           "66.189.127/27",
-          "68.124.42.162/27"
+          "68.184.42.162/27"
         ]
       },
       "description": "Using subnet mask these ranges cover 66.189.127.0-32 and 68.124.42.162.160-191"
```
Comment (danqrodney) [inline rest-api/config/config_prod.json]: Fix the description, too?

File: rest-api/config/config_stable.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
         "roles": ["awardee_sa"],
         "awardee": "TRANS_AM",
         "whitelisted_ip_ranges" : {
-          "ip4": ["66.189.127/27", "68.124.42.162/27"]
+          "ip4": ["66.189.127/27", "68.184.42.162/27"]
         },
         "description": "Using subnet mask these ranges cover 66.189.127.0-32 and 68.124.42.162.160-191"
```
Comment (danqrodney) [inline rest-api/config/config_stable.json]: Same here

File: rest-api/config/config_prod.json (hunk 1 above)
Comment (Seeker1911) [reply rest-api/config/config_prod.json]: done

File: rest-api/config/config_stable.json (hunk 2 above)
Comment (Seeker1911) [reply rest-api/config/config_stable.json]: done

**Pull request merged**